### ISSUE TITLE
Code not published but paper is out?

### ISSUE THREAD

**Issue by andreped** [wujiong-hub/LSIL#1]
Just came across [your paper](https://link.springer.com/article/10.1007/s11548-022-02788-9) on volumetric medical image registration, which I found extremely interesting.

As I wanted to play around with the idea, I went over to the provided GitHub URL.

Sadly, I observed that the source code had not yet been released. Could you make it public?

I will be testing it on a different application, but would be great if it did not require that much work. Hence, please, share a description on how to use the code, including a `requirements.txt` file as well which should include dependencies and whatnot.

In advance, thanks! :]

**Comment by jpdefrutos** [comment]
I agree with @andreped. Personally, I find your work very relevant and it would be very helpful to have access to your code @wujiong-hub 😊

Much appreciated!
